src/index.ts

```diff
@@ -219,13 +219,15 @@ async function dockerBuild(tag: string, args: string[]): Promise<number> {
     dockerArgs.push('--entrypoint', '/bin/bash')
   }
 
-  core.startGroup('Pull Docker image')
-  core.info(`Using ${image} Docker image`)
-  const exitCode = await exec.exec('docker', ['pull', image])
-  if (exitCode !== 0) {
-    throw new Error(`maturin: 'docker pull' returned ${exitCode}`)
+  if (container.includes('/')) {
```

Comment:
I think this can be checked better with `docker images -q $image`, see https://stackoverflow.com/a/30543453